docs/mkdocs/docs/tutorials/data_organisation.md

```diff
@@ -0,0 +1,233 @@
+# Guidelines for Organising Your Data
+
+## Introduction
+
+We are often asked about how best to organise data in ArcticDB. There is no single answer to this question - it will depend on many factors, but especially on the data itself and how it will be used.
+
+ArcticDB offers a high degree of flexibility in how data is arranged. Although this is a strength of the system, sometimes the choices are so broad that it is hard to know where to start.
+
+In this guide we aim to outline some design principles and rules of thumb to help you decide the best scheme for your data, system and users.
+
+We begin by taking a design view of the way ArcticDB structures data and the system design concerns, then from those consider how to assess and balance the trade-offs of some typical organisation strategies.
+
+## ArcticDB's Data Hierarchy
+
+Let's revisit the structures that ArcticDB provides for organising data, illustrated below
+
+![](../images/ArcticDBDataHierarchy.png)
+
+### Object Store
+
+The object stores available will typically be decided by external factors such as
+
+* Environment: prod/uat/research
+* Permissions: enviroments that grant/deny read/write permission to different groups of users/systems
+* Accounting: different cloud buckets may be charged to different internal accounts
+* Storage Quotas: different amounts of storage may be allocated for different purposes
+* Storage Performance: faster/slower storage for different applications according to requirements and cost
+
+
+### Library
+
+ArcticDB can handle a large number of libraries per object store - up to 100,000 is perfectly reasonable.

```

Comment:
Have we tested that the performance of this is acceptable for operations like `list_libraries`?